web/client/components/data/query/GeometryDetails.jsx

```diff
@@ -168,28 +169,44 @@ class GeometryDetails extends React.Component {
     };
 
     onClosePanel = () => {
-        if (this.props.type === "BBOX") {
-            this.resetBBOX();
-        } else if (this.props.type === "Circle") {
-            this.resetCircle();
-        }
-
+        this.resetGeom();
         this.props.onShowPanel(false);
     };
-
+    roundValue = (val, prec = 1000000) => Math.round(val * prec) / prec;
+    getStep = (zoom = 1) => {
+        if ( zoom >= 21 ) {
+            return 0.00001
+        }else if( zoom >= 18) {
+            return 0.0001
+        }else if( zoom >= 15) {
+            return 0.001
+        }else if( zoom >= 12) {
+            return 0.01
+        }else if( zoom >= 9) {
+            return 0.1
+        }else if( zoom >= 6) {
+            return 1
+        }
+            return 10;
+    };
+    getStepCircle = (zoom, name) => {
+        const step = this.getStep(zoom);
+        return name === 'radius' && step * 100000 || step;
+    };
+    isWGS84 = () => (this.props.geometry || {}).projection === 'EPSG:4326' || !this.props.useMapProjection;
```

Comment:
I think we can use proj4 to know the uom of the projection, and generalize EPSG:4326 to uom === 'degrees'